lib/ansible/config/manager.py

```diff
@@ -90,104 +92,108 @@ def ensure_type(value, value_type, origin=None, origin_ftype=None):
             tildes's in the value.
         :str: Sets the value to string types.
         :string: Same as 'str'
+    :kwarg origin: String representing where the value came from ('default', 'env', 'vars', etc), can be a file path
+    :kwarg origin_ftype: If the origin is a config file, the type, currently only 'ini' and 'yaml'
```

Comment:
There's no such thing as `kwarg`. The correct syntax is.
```suggestion
    :param origin: String representing where the value came from ('default', 'env', 'vars', etc), can be a file path
    :param origin_ftype: If the origin is a config file, the type, currently only 'ini' and 'yaml'
```

Reply:
i was following the existing 'standard', this was something we considered at one point when there were multiple competing ones, since we have not yet decided on a formal standard for core, i'll just add to existing instead of rewriting.